server/ansible/roles/pbench-server-user/tasks/main.yml

```diff
@@ -27,3 +27,12 @@
     owner: "{{ pbench_user }}"
     group: "{{ pbench_group }}"
     state: directory
+
+- name: set SELinux context on /home/<pbench>/.ssh
+  sefcontext:
```

Comment:
`s/sefcontext/setcontext/`?

Reply:
No, the module is called [sefcontext](https://docs.ansible.com/ansible/latest/modules/sefcontext_module.html).